apps/api-extractor/src/analyzer/ExportAnalyzer.ts

```diff
@@ -155,10 +155,22 @@ export class ExportAnalyzer {
           if (exportStarSymbol) {
             for (const exportStarDeclaration of exportStarSymbol.getDeclarations() || []) {
               if (ts.isExportDeclaration(exportStarDeclaration)) {
-                const starExportedModule: AstModule | undefined = this._fetchSpecifierAstModule(
-                  exportStarDeclaration,
-                  exportStarSymbol
-                );
+                let starExportedModule: AstModule | undefined;
+
+                try {
+                  starExportedModule = this._fetchSpecifierAstModule(exportStarDeclaration, exportStarSymbol);
+                } catch {
```

Comment:
Is this catching `Internal Error: getResolvedModule() could not resolve module name`?  That error is thrown by API Extractor's own implementation.  Generally we try to avoid throwing/catching exceptions during normal successful operation.

Reply:
Yes, I explained in the PR description:
> Why use `try...catch`? The `TypeScriptHelpers.isAmbient` cannot working for this, is there a more precise way than `try...catch`?

Can you provide a better way to catch that?